docs/source/faq.rst

```diff
@@ -0,0 +1,154 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page contains frequently asked questions about MOTECH with short answers to them.
+
+.. contents::
+    :depth: 2
+
+Can MOTECH run on Windows?
+--------------------------
+MOTECH is available and can be deployed on Windows, but it is not actively supported.
+
+Is MDS using any caching? How does that affect a cluster deployment?
+--------------------------------------------------------------------
+MDS is using the DataNucleus persistence framework, which by default uses level1 and level2 cache. It can cause data inconsistency
+across multiple server nodes in cluster deployment. To resolve this issue you can turn off level2 cache by setting
+``datanucleus.cache.level2.type`` property to ``none``. There may be slight performance drop per server in result of
+turning off level2 cache. Level1 cache should be kept as it works within one transaction, therefore not affecting
+cluster deployment.
+
+More information about the DataNucleus can be found on it project page (link_). There is also available caching doc_.
+
+.. _link: http://www.datanucleus.org/
+.. _doc: http://www.datanucleus.org/products/accessplatform_4_0/jpa/cache.html
+
+How to use SSL for database connections in MOTECH?
+--------------------------------------------------
+First generate and install your SSL certificates. After this is done you will need to configure your database to use SSL.
+For MySQL you can follow instructions_ from official MySQL page.
+Next step is to point JVM variables to your SSL certificates. Look here_ for more information about these variables.
```

Comment:
Additionaly, change "here" to something meaningful, so we don't end up with hundreads of "here" links. Perhaps "Look at the Oracle documentation for more info"~